src/navigation/rootNavigation.js

```diff
@@ -32,6 +36,10 @@ const hideHeader = {
   headerShown: false
 };
 
+const queryClient = new QueryClient( {
+  defaultOptions: { queries: { retry: 2 } }
```

Comment:
Probably fine for now, but eventually we'd want retry to be a function that only retries errors that look like timeouts. Most HTTP errors should be idempotent.